public/models/harvard_aeon/archival_object_mapper.rb

```diff
@@ -8,6 +8,20 @@ def map_extensions(mapped, item, repository, resource, resource_json)
       mapped.ext(:level).name = item['level'].capitalize
     end
 
+    def request_permitted?(item)
+      Rails.logger.error(item['json']['jsonmodel_type'])
+      if item['json']['jsonmodel_type'] == "archival_object" and item['json']['instances'] and item['json']['instances'].length >= 1
+        item['json']['instances'].each do |my_instance|
+          if my_instance['instance_type'] and my_instance['instance_type'] == "text" and my_instance['subcontainer'] and my_instance['subcontainer']['topcontainer'] and my_instance['subcontainter']['topcontainer']['ref']
+            true
+          else
+            false
+          end
+        end
+      else
+        false
+      end  
```

Comment:
it makes more sense to return false outside the top-most if block so it catches everything
```suggestion
      end 
      return false
```